frontend/src/components/alert/alert-styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+import { AlertBoxProp } from '../../interfaces/styles';
+import { ButtonProps } from '../../interfaces/types';
+import { theme } from '../../theme';
+
+export const AlertBox = styled.div<AlertBoxProp>`
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  display: flex;
+  width: 500px;
+  height: 100px;
+  border-radius: 10px;
+  box-shadow: 0px 0px 8px 0px ${theme.colors.shadow};
+  color: ${({ type }) => {
+    if (type === 'error') {
+      return theme.colors.danger;
+    }
+    if (type === 'success') {
+      return theme.colors.success;
+    }
+    return theme.colors.info;
+  }};
+
+  border: 1px solid
+    ${({ type }) => {
+      if (type === 'error') {
+        return theme.colors.danger;
+      }
+      if (type === 'success') {
+        return theme.colors.success;
+      }
+      return theme.colors.info;
+    }};
+  background-color: white;
+`;
+
+export const ContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+export const Title = styled.div`
+  margin-bottom: ${theme.space[2]};
+  font-size: ${theme.fontSizes[4]};
```

Comment:
Think about refactoring theme.space[2] or theme.fontSizes[4] ...this is to abstract. 
It would be much readable and clear if you use objects instead of arrays. For example theme.fontSize.xl

Reply:
We have too many variants of those to give them good enough aliases

Reply:
Then you shouldn't have that many :)

Reply:
ok, this issue will be fixed in refactor branch